tools/errortracker/errortracker.go

```diff
@@ -174,9 +175,15 @@ func handle(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	exception := r.URL.Query().Get("s")
+	// If format does not end with :\d+ truncate up to the last newline.
+	if !regexp.MustCompile(`:\d+\n?$`).MatchString(exception) {
+		exception = string(regexp.MustCompile(`\n.*$`).ReplaceAllString(exception, ""))
```

Comment:
<img width="941" alt="screen shot 2017-05-09 at 5 02 01 pm" src="https://cloud.githubusercontent.com/assets/354746/25877595/44bec5d6-34d9-11e7-9c5b-0b1945d91c24.png">

@dvoytenko  can you sanity check for me?